cmd/cueckoo/cmd/importpr.go

```diff
@@ -209,9 +209,13 @@ func importPRDef(c *Command, args []string) error {
 
 func run(ctx context.Context, name string, args ...string) (string, error) {
 	cmd := exec.CommandContext(ctx, name, args...)
-	out, err := cmd.CombinedOutput()
+	out, err := cmd.Output()
 	if err != nil {
-		return "", fmt.Errorf("%v: %s", err, out)
+		if err, _ := err.(*exec.ExitError); err != nil {
+			// stderr was captured by Output
```

Comment:
Not sure this is the correct comment. Output() only captures stdout, it's just that (*exec.ExitError).Stderr is set if cmd.Stderr is nil

https://pkg.go.dev/os/exec#Cmd.Output

Reply:
Right, what's what I meant. We don't set Stderr, so the Output method captured stderr for us.